lib/tran_pipe.c

```diff
@@ -250,14 +250,12 @@ tran_pipe_init(vfu_ctx_t *vfu_ctx)
     tp = calloc(1, sizeof(tran_pipe_t));
 
     if (tp == NULL) {
-        ret = errno;
-        goto out;
+        return -1;
```

Comment:
return ERROR_INT(errno)

Reply:
why? What's here is the exact same.

Reply:
Just to be consistent with other returns. But I am fine with this also.